src/gui/sharedialog.cpp

```diff
@@ -35,6 +36,7 @@
 #include <QPointer>
 #include <QPushButton>
 #include <QFrame>
+#include <iostream>
```

Comment:
are you sure you need it ?

Reply:
Nope, sorry, was using for debugging. removed